src/app/app-auth.service.ts

```diff
@@ -18,13 +18,13 @@ import {Location, PopStateEvent} from '@angular/common';
 @Injectable()
 export class AppAuthService {
   // isRequired = true;
-  public trustedURLs = new Set();
+  public trustedURLs: Set<string> = new Set();
   // location: any;
   public currentUrl: string = null;
 
   constructor(private location: Location) {
     // TODO this should use the observable and expose an observable.
-    location.subscribe((popStateEvent: PopStateEvent) => {
+    this.location.subscribe((popStateEvent: PopStateEvent) => {
```

Comment:
If location is not used anywhere else, you could just remove `private` in https://github.com/DD-DeCaF/caffeine/pull/9/files#diff-ea4ad94cd4b564ad0b7412935abb8c2fR25